Update testcafe config, ignore prefer-default-export rule (#4)
DevExpress/eslint-config-devextreme: README.md

  - [`devextreme/typescript`](#typescript)
  - [`devextreme/spell-check`](#spell-check)

- **Overrides**

>   - Tests can use any type of dependencies.

Reviewer: ```suggestion
  - Allow using any types of dependencies:
```

    ```javascript
    'import/no-extraneous-dependencies': 'off'
    ```

>   - We should test event handlers that use modified params to work.

Reviewer: ```suggestion
  - Allow modifying parameters in event handlers to test these use cases:
```

  
    ```javascript
    'no-param-reassign': ['error', { 'props': false }],
    ```

>   - It's a common case for the TestCafe tests that does not affect the performance.

Reviewer: ```suggestion
- Allow `await` inside of loops. This is a common case for TestCafe tests.
```
